Mathlib/Algebra/Order/Monoid/Unbundled/Basic.lean

```diff
@@ -325,6 +343,18 @@ lemma min_mul [CovariantClass α α (swap (· * ·)) (· ≤ ·)] (a b c : α) :
   haveI := mulRightMono_of_mulRightStrictMono α
   Left.min_le_max_of_mul_le_mul h
 
+@[to_additive] lemma Left.mul_left_cancel_of_mulLeftStrictMono [MulLeftStrictMono α]
```

Comment:
Should this be an instance of `AddLeftCancel`?

Reply:
I thought that I was following your suggestion [here](https://github.com/leanprover-community/mathlib4/pull/6627#discussion_r1925340721) to not make it an instance.  I am happy to make it an instance, if you prefer!